### ISSUE TITLE
Better toggling

### ISSUE THREAD

**Issue by quick007** [jjranalli/nightwind#19]
I made an issue a few days ago asking some questions, and I really think something like this would help with bigger, account-based applications:

```ts
nightwind.toggle(true)
//toggles nightwind on

nightwind.toggle(false)
//toggles nightwind off
```

**Comment by Djiit** [comment]
I think toggle is intended to be switching on and off, depending on the current state. Maybe we could add setDarkMode/setLightMode ? Added a PR with a proposed interface.

**Comment by Djiit** [comment]
@quick007 is it better now that we merged the PR ?

**Comment by jjranalli** [comment]
@Djiit The PR was merged into a separate branch. It will get merged with the main branch on the next major v1.2.0 release! Hopefully soon :)

**Comment by quick007** [comment]
I'm going to go ahead and close this because of #21

**Comment by jjranalli** [comment]
Closing as the issue has been fixed in 1.1.12
